src/shared/selinux-util.c

```diff
@@ -143,15 +149,20 @@ static int open_label_db(void) {
 }
 #endif
 
-int mac_selinux_init(void) {
+static int selinux_init(bool is_lazy) {
```

Comment:
Maybe, better to rename `is_lazy` -> `force` and inverse the flag?
I spent a few minutes to understand the meaning of the flag...